lib/rspec/mocks/message_expectation.rb

```diff
@@ -32,6 +32,8 @@ def verify_messages_received
           end
         end
       end
+
+      def unadvise(_) ; end
```

Comment:
I went off this guide (https://github.com/bbatsov/ruby-style-guide#no-single-line-methods), but  consistency is key, so it would make sense to match existing style.

Something like this?

    def unadvise(*_args)
    end